scripts/build-docs.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Would you mind if I took a stab at writing this as a node script?  I think they're a little more powerful/efficient and maintainable overall.